contracts/QuestFactory.sol

```diff
@@ -170,6 +170,16 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         return quests[questId_].numberMinted;
     }
 
+    /// @dev return data in the quest struct for a questId
+    /// @param questId_ The id of the quest
+    function questInfo(string memory questId_) external view returns (address, uint, uint) {
+        return (
+            quests[questId_].questAddress,
+            quests[questId_].totalParticipants,
+            quests[questId_].numberMinted
+        );
```

Comment:
I like this! But haven't really worked with tuples on the ethers.js side so interesting to see how the data will return (maybe a JS object?)

Reply:
Nvm I see this mostly for RabbitHoleReceipt.sol